aliases.sh

```diff
@@ -22,6 +22,7 @@ alias reminderbatch='python -m reminder_batch_scheduler.reminder_batch'
 alias bulkrefusals='python -m bulk_processing.refusal_processor'
 alias bulknewaddresses='python -m bulk_processing.new_address_processor'
 alias bulkinvalidaddresses='python -m bulk_processing.invalid_address_processor'
+alias caseidaddressdelta='invalid_address_delta.sh'
```

Comment:
Alias should reflect it's for invalid address delta e.g. `alias invalidaddressdelta`. There are a few types of address delta, worth avoiding any confusion in the tool names.